gradle.properties

```diff
@@ -31,7 +31,7 @@ KOTLINX_COLLECTIONS_IMMUTABLE_VERSION=0.1
 KOTLINX_COROUTINES_VERSION=1.3.3
 KOTLINX_VALIDATOR_VERSION=0.2.2
 KTLINT_GRADLE_VERSION=9.2.1
-KTLINT_VERSION=0.34.2
+KTLINT_VERSION=0.36.0
```

Comment:
It's using Kotlin 1.3.60... Isn't it necessary to change KTLINT GRADLE VERSION?

Reply:
I checked that out.. and 9.2.1 is already the latest version

Reply:
Great! Why are we changing this version if it's using 1.3.60? Could its dependencies be managed?

Reply:
its using 1.3.41:
```
     +--- com.pinterest.ktlint:ktlint-core:0.34.2
     |    \--- org.jetbrains.kotlin:kotlin-compiler-embeddable:1.3.41
     |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.3.41 (*)
     |         +--- org.jetbrains.kotlin:kotlin-script-runtime:1.3.41
     |         +--- org.jetbrains.kotlin:kotlin-reflect:1.3.41
     |         |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.3.41 (*)
```
so we can update the version.. or manage its dependencies.. whats better here ?

Reply:
Right, and `0.36.0` is using Kotlin 1.3.60. I think we're in the same situation. I'd manage its dependencies.

Reply:
okay, will do it like that 👍 thanks @rachelcarmena !! 😄